app/models.py

```diff
@@ -1448,6 +1485,8 @@ def status(self):
     def status(cls):
         return sql_case([
             (cls.submitted_at.isnot(None), 'submitted'),
+            (cls.awarded_at.isnot(None), 'awarded'),
```

Comment:
I think this uses the python version of the hybrid property to get the status. I'm not sure of the best way to test the sql version.